src/test/resources/features/starting-world.feature

```diff
@@ -0,0 +1,21 @@
+Feature: As a user I should be able see a textual output of the game of life world
```

Comment:
Actually, if this is simply a test that the previously inputted grid representation results to a correct output representation. Then I would presonally prefer that it didn't increment the generation. 

Simply a test that does:
```
internalDataRep = Deserialize(input);
output = Serialize(internalDataRep);
assertEquals(output,input)
```

Reply:
fixed